controllers/front/OrderConfirmationController.php

```diff
@@ -81,11 +81,14 @@ public function init()
     public function initContent()
     {
         parent::initContent();
+        $order = new Order(Order::getOrderByCartId((int)($this->id_cart)));
 
         $this->context->smarty->assign(array(
             'is_guest' => $this->context->customer->is_guest,
             'HOOK_ORDER_CONFIRMATION' => $this->displayOrderConfirmation(),
-            'HOOK_PAYMENT_RETURN' => $this->displayPaymentReturn()
+            'HOOK_PAYMENT_RETURN' => $this->displayPaymentReturn(),
+            'url_to_invoice' => HistoryController::getUrlToInvoice($order, $this->context),
+            'products' => $order->getCartProducts()
```

Comment:
Please remove this for now. We need the whole order but we don't have an easy way to present it yet.